deltacat/utils/metrics.py

```diff
@@ -117,8 +128,51 @@ def _emit_cloudwatch_metrics(
         )
 
 
+@metric_scope
+def _emit_cloudwatch_emf_metrics(
+    metrics: MetricsLogger,
+    metrics_name: str,
+    metrics_type: Enum,
+    metrics_config: MetricsConfig,
+    value: str,
+) -> None:
+    ct = datetime.now()
+    dimensions = dict(
+        [(dim["Name"], dim["Value"]) for dim in metrics_config.metrics_dimensions]
+    )
+    metrics_name_with_type = _build_metrics_name(metrics_type, metrics_name)
+    try:
+        metrics.set_timestamp(ct)
+        metrics.set_dimensions(dimensions)
+        metrics.set_namespace(metrics_config.metrics_namespace)
+
+        metrics_kwargs = metrics_config.metrics_kwargs
+
+        Config = get_config()
```

Comment:
Config is a class?

Reply:
Are you saying it should be renamed to something more generic?